Annotation/constituent_golden_set.json

```diff
@@ -419,7 +486,7 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "how much he has grown",

```

Comment:
Again, a degree clause so I'm not sure if this is truly an interrogative embedded clause